### ISSUE TITLE
Button Description Accessibility

### ISSUE THREAD

**Issue by jolanglinais** [accordproject/template-studio-v2#40]
# WCAG 2.0 Level A [2.4.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html) (Link Purpose)

## Tasks
- Ambiguous buttons should be more descriptive for screen readers. This can be achieved by creating a visually hidden span with a description in it.

## Related Issues: 
- Issue [#55](https://github.com/accordproject/cicero-ui/issues/55) in Cicero-UI

**Comment by nicoleCamoro** [comment]
Hello, I'll take this task. :)

**Comment by Sweta271097** [comment]
@irmerk Since there has not been a PR for 7 days when it was assigned, do you think maybe is it okay for me to work on this issue?

**Comment by jolanglinais** [comment]
Moving this back to open after 7 days.
